Jellyfin.Data/Jellyfin.Data.csproj

```diff
@@ -0,0 +1,46 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
```suggestion
    <TargetFramework>netstandard2.1</TargetFramework>
    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Reply:
`MediaBrowser.Model` targets 2.0 and 2.1, so either this would have to target both as well, or I could make `MediaBrowser.Model` only target 2.1

Reply:
I updated both `MediaBrowser.Model` and `Jellyfin.Data` to netstandard 2.1, there were no problems in my testing.